jupyter-docker/spark_install_hail.sh

```diff
@@ -29,7 +29,7 @@ spark.submit.pyFiles ${HAIL_HOME}/${HAILZIP},${HAIL_HOME}/${HAILPYTHON}
 spark.driver.extraClassPath ${HAIL_HOME}/${HAILJAR}
 
 # Add JARs to Classpaths: distributed to executor working directory by above spark.jars directive
-spark.executor.extraClassPath ${HAIL_HOME}/${HAILJAR}
+spark.executor.extraClassPath ./${HAILJAR}
```

Comment:
This is the hail bug! The `spark.driver.extraClassPath` is used on the master node, which is where we put the Hail jar, so it should be an absolute path. The `spark.executor.extraClassPath` is used on the workers where everything is put into a flat directory, so it needs to be a _relative_ path.

Thanks to Cotton who saw this immediately, I don't know how I would have figured it out myself.

Reply:
oof